.env.template

```diff
@@ -15,5 +15,8 @@ export AFFILS_AWS_COGNITO_APP_CLIENT_ID="placeholder"
 export AFFILS_AWS_COGNITO_APP_CLIENT_SECRET="placeholder"
 export AFFILS_AWS_REGION="placeholder"
 export AFFILS_AWS_SECRET_KEY="placeholder"
+export AFFILS_AWS_BUCKET_NAME="placeholder"
```

Comment:
I would rename these to:

`AFFILS_AWS_S3_BUCKET_NAME`
`AFFILS_AWS_S3_ACL`

If they are indeed related to an S3 bucket. If these are related to a Lightsail bucket, I would rename them:

`AFFILS_AWS_LIGHTSAIL_BUCKET_NAME`
`AFFILS_AWS_LIGHTSAIL_ACL`

Reply:
Good idea! I have made that change.